farm_2side.json

```diff
@@ -0,0 +1,213 @@
+        " ": [ "t_grass_long", "t_grass", "t_grass", "t_grass", "t_dirt" ],
+        "Q": [ "t_grass", "t_grass", "t_grass", "t_dirt" ],
+        "#": [ "t_wall_wood" ],
+        "I": [ "t_wall_metal" ],
+        "i": [ "t_metal_floor" ],
+        "$": [ "t_shrub" ],
+        "~": [ "t_water_sh" ],
+        "t": [ "t_tree" ],
+        "&": [ "t_water_pump" ],
+        "s": [ "t_sidewalk" ],
+        "+": [ "t_door_c" ],
+        ",": [ "t_dirt" ],
+        "-": [ "t_wall" ],
+        ".": [ "t_floor" ],
+        "'": [ "t_linoleum_white" ],
+        "1": [ "t_column" ],
+        "5": [ "t_palisade_pulley" ],
+        "B": [ "t_floor" ],
+        "D": [ "t_dirtmound" ],
+        "E": [ "t_dirtmoundfloor" ],
+        "F": [ "t_fence_v" ],
+        "H": [ "t_floor" ],
+        "M": [ "t_door_metal_locked" ],
+        "O": [ "t_barndoor" ],
+        "P": [ "t_grass" ],
+        "S": [ "t_linoleum_white" ],
+        "T": [ "t_linoleum_white" ],
+        "W": [ "t_window_boarded" ],
+        "Y": [ "t_floor" ],
+        "_": [ "t_dirtfloor" ],
+        "b": [ "t_linoleum_white" ],
+        "d": [ "t_floor" ],
+        "e": [ "t_linoleum_white" ],
+        "g": [ "t_fencegate_c" ],
+        "h": [ "t_floor" ],
+        "n": [ "t_floor" ],
+        "k": [ "t_floor" ],
+        "l": [ "t_floor" ],
+        "o": [ "t_linoleum_white" ],
+        "u": [ "t_floor" ],
+        "w": [ "t_window_domestic" ],
+        "y": [ "t_dirtfloor" ],
+        "|": [ "t_wall" ]
+      },
+      "furniture": {
+        "B": [ "f_bed" ],
+        "H": [ "f_armchair" ],
+        "P": [ "f_mailbox" ],
+        "S": [ "f_sink" ],
+        "T": [ "f_toilet" ],
+        "Y": [ "f_rack_coat" ],
+        "b": [ "f_bathtub" ],
+        "d": [ "f_dresser" ],
+        "e": [ "f_fridge" ],
+        "h": [ "f_chair" ],
+        "n": [ "f_table" ],
+        "k": [ "f_desk" ],
+        "l": [ "f_locker" ],
+        "o": [ "f_oven" ],
+        "u": [ "f_cupboard" ],
+        "y": [ "f_hay" ]
+      },
+      "toilets": { "T": {  } },
+      "items": {
+        "B": { "item": "bed", "chance": 60 },
+        "D": { "item": "hydro", "chance": 5 },
+        "H": { "item": "livingroom", "chance": 65 },
+        "b": { "item": "softdrugs", "chance": 70 },
+        "d": [ { "item": "dresser", "chance": 80 }, { "item": "homeguns", "chance": 20 } ],
+        "e": { "item": "fridge", "chance": 65 },
+        "h": { "item": "bedroom", "chance": 65 },
+        "k": { "item": "bedroom", "chance": 65 },
+        "Y": { "item": "coat_rack", "chance": 35, "repeat": [ 1, 4 ] },
+        "P": { "item": "mail", "chance": 30, "repeat": [ 2, 5 ] },
+        "l": [
+          { "item": "tools_earthworking", "chance": 65 },
+          { "item": "mechanics", "chance": 40 },
+          { "item": "home_hw", "chance": 50 },
+          { "item": "tools_common", "chance": 50 }
+        ],
+        "u": [
+          { "item": "kitchen", "chance": 70 },
+          { "item": "cannedfood", "chance": 65 },
```

Comment:
Overall I'm working on increasing some types of item spawns significantly, to indicate the game start time as only a few days after the end of the world. I'd suggest you could add some repeats to several of these thngs like fridge, bedroom, and cannedfood if you want.